CHANGELOG.md

```diff
@@ -11,13 +11,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - CODE_OF_CONDUCT.md, CONTRIBUTING.md
 - `getHtml()` and `setHtml()` APIs. [#112](https://github.com/graphcool/chromeless/pull/112), [#74](https://github.com/graphcool/chromeless/issues/74)
 - `mousedown(selector: string): Chromeless<T>` and `mouseup(selector: string): Chromeless<T>` APIs [#118](https://github.com/graphcool/chromeless/pull/118) @criticalbh
+- `cookiesGet(name: string)` [#128](https://github.com/graphcool/chromeless/pull/128) API @criticalbh
```

Comment:
as I could understand, if we want to take all cookies, we will use `cookies` getter, but if we are going to query via name or query object than we would use method `cookiesGet(name | query)`?